collection-scripts/gather_logs

```diff
@@ -34,13 +34,43 @@ for ns in ${namespaces[@]}; do
     done
   done
 
+  for migplan in $(oc get migplan --namespace ${ns} --no-headers | awk '{print $1}'); do
+    echo ${migplan}
+    mkdir -p "/must-gather/migplan-${migplan}"
+    oc get migplan ${migplan} -o yaml --namespace ${ns} &> "/must-gather/migplan-${migplan}/${migplan}.yml"
+  done
+
+  for migmigration in $(oc get migmigration --namespace ${ns} --no-headers | awk '{print $1}'); do
+    echo ${migmigration}
+    migplan=$(oc get migmigration ${migmigration} --namespace ${ns} -o jsonpath='{.spec.migPlanRef.name}')
```

Comment:
Disregard previous comment, I missed what you're doing here. This looks like it should work.